src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs

```diff
@@ -223,8 +257,13 @@ private static bool ContainsMultipleTypesInSpine(SyntaxNode node)
 
         private static string GetNewSymbolName(ISymbol symbol, string targetNamespace)
         {
-            Debug.Assert(symbol != null);
-            return targetNamespace + symbol.ToDisplayString().Substring(symbol.ContainingNamespace.ToDisplayString().Length);
+            Debug.Assert(symbol != null && !string.IsNullOrEmpty(targetNamespace));
+
+            var offset = symbol.ContainingNamespace.IsGlobalNamespace
+                ? 0
+                : symbol.ContainingNamespace.ToDisplayString().Length + 1;
```

Comment:
if you're trimming off the name of hte containing namespace... then isn't this just `symbol.Name`?  or `symbol.ToDisplayString(option that says you don't want namespaces)`?